src/skin/options.html

```diff
@@ -194,6 +194,17 @@ <h3><span class="i18n_export_user_data"></span></h3>
         <button id="exportTrackers"><span class="i18n_download"></span></button>
       </div>
     </div>
+    <div class="btn-silo">
+      <p class="i18n_sync_intro"></p>
```

Comment:
Could we fix margin/padding around this paragraph to be a bit more balanced? It feels like there is too much space at the bottom and too little at the top.

Reply:
I added a bit at the top and took off all the margin at the bottom, but the padding/margin on the div/h3 below it is responsible for ~2/3 of that space.  I'd prefer not to make those special for this row.